unfollow/config.py

```diff
@@ -24,14 +24,16 @@
 # text for each language for each theme
 [locale.english]
 [locale.english.regular]
-welcome_message = ":dancer: [purple]Welcome to[/purple] [red]who-unfollowed-me[/red][blue] Python implementation[/blue] by [#FFD700]Zac the Wise[#FFD700]"
+welcome_message = ":dancer: [purple]Welcome to[/purple] [red]who-unfollowed-me[/red][blue] Python implementation[/blue]
+by [#FFD700]Zac the Wise[#FFD700]"
```

Comment:
I believe these should be just one line as it was before, or multi-line using three quotation marks instead of one.

Reply:
Well using """ quotes would make it as a comment and not a string and it was crossing the line character length so i made it as two line comments, so what do u recommend to do ?

Reply:
Just put it back to one and don't worry too much about the line too long issue, the CI will no longer fail because of it

Reply:
fixed it